tests/testthat/reporters/check.txt

```diff
@@ -18,10 +18,18 @@ stop
 
 6. Error: Error:3 (@tests.R#36) ------------------------------------------------
 !
-1: f() at reporters/tests.R:36
-2: g() at reporters/tests.R:32
-3: h() at reporters/tests.R:33
-4: stop("!") at reporters/tests.R:34
+1: expect_message(f(), NA) at reporters/tests.R:36
```

Comment:
You'll also need to tweak the constants in `test_code()` to avoid the extra stuff here